src/redux/books/books.js

```diff
@@ -0,0 +1,28 @@
+const ADD_BOOK = 'redux/books/book/ADD_BOOK';
+const REMOVE_BOOK = 'redux/books/book/REMOVE_BOOK';
+
+const initialState = [];
+
+const newBook = (id, title, author) => ({
+  type: 'ADD_BOOK',
+  title,
+  author,
+});
```

Comment:
- [ ] We would need to save the `id` as well. Kindly include it as part of the returned attribute:
  ```
  const newBook = (id, title, author) => ({
    type: 'ADD_BOOK',
    id,
    title,
    author,
  });
  ```
  or better yet you can do this:
  ```
  const newBook = (payload) => ({
    type: 'ADD_BOOK',
    payload
  });
  ```